lib/matplotlib/tests/test_axes.py

```diff
@@ -4673,10 +4667,7 @@ def test_pandas_errorbar_indexing():
 
 
 def test_pandas_indexing_hist():
-    try:
-        import pandas as pd
-    except ImportError:
-        pytest.skip("Pandas not installed")
+    pd = pytest.importorskip('pandas')
```

Comment:
Is there a minimum required version of pandas needed for the pandas tests to work?